mlrun/wasbfs/__init__.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""wasbfs - An fsspec driver for wasb/wasbs"""
```

Comment:
I think `wasbfs` code should be inside the `mlrun/datastore` directory like all other datastore implementations.

Reply:
wasbfs is implementation of fs and not datastore. the equivalent fs's are implemented in different repo.
in this case there is no justification to do so.

Reply:
moved